rpc/handlers_test.go

```diff
@@ -1843,16 +1843,19 @@ func TestTransactionStatus(t *testing.T) {
 		network           utils.Network
 		verifiedTxHash    *felt.Felt
 		nonVerifiedTxHash *felt.Felt
+		notFoundTxHash    *felt.Felt
```

Comment:
nit: can we move the "not found" case to a separate sub-test? That should reduce the complexity of the test.

Reply:
@joshklop 
I Have moved the "not found" case to a separate sub-test